src/meteofrance/helpers.py

```diff
@@ -2,7 +2,9 @@
 """Météo-France helpers."""
 import math
 from datetime import datetime
+from typing import Dict
 from typing import List
+from typing import Optional
 from typing import Tuple
```

Comment:
I would prefer one line import, same bellow.

Reply:
I use [reorder-python-imports](https://pypi.org/project/reorder-python-imports/) to check and order automatically the import. They split imports to reduce merge conflicts.
I want to continue this way.